### DISCUSSION TITLE
Are cross-org workflow_dispatch event unsupported? Or is there a way to do this?

### DISCUSSION THREAD

**Discussion by franklywatson** [community/community#36480] [Actions]
I have successfully created a GH Action that triggers a workflow_dispatch event to another repo in the same GH org. However, I wanted to also be able to trigger that same WF from another org using a workflow_dispatch trigger. 

I've followed the process to create a GH App which is applied to both the orgs in question. Having ensured that my token is not a PAT and is provided through the GH app i get smooth triggering between repos in the same org. 

When I call the API on the other org repo i get 404, for the same code that worked above. Obviously that's some permissions scope thing. When i looked into it it seems Github Apps offers only the following [permissions](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#actions):

```
GET /repos/:owner/:repo/actions/workflows (read)
GET /repos/:owner/:repo/actions/workflows/:workflow_id (read)
GET /repos/:owner/:repo/actions/workflows/:workflow_id/runs (read)

```

However it seems like no write, which workflow_dispatch [requires](https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event)

Could someone on staff confirm if this is accurate and it's not permitted at all? Or otherwise would appreciate knowing what I should do if it is in fact possible? Thanks

**Comment by ossanna16** [comment]
Hey there @franklywatson and thank you for asking a good question! I'm glad you found our community 🙂
